sentry-core/src/main/java/io/sentry/core/transport/HttpTransport.java

```diff
@@ -30,14 +30,15 @@
 import java.util.Date;
 import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
+import java.util.zip.GZIPOutputStream;
 import javax.net.ssl.HttpsURLConnection;
 import org.jetbrains.annotations.ApiStatus;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
 /**
  * An implementation of the {@link ITransport} interface that sends the events to the Sentry server
- * over HTTP(S) in UTF-8 encoding.
+ * over HTTP(S).
```

Comment:
`l`: is it worth mentioning here that we use gzip?

Reply:
it was `gzip`, but @rhcarvalho preferred to not mention at all.